pyaedt/application/Design.py

```diff
@@ -2499,11 +2487,11 @@ def evaluate_expression(self, expression_string):
     def design_variation(self, variation_string=None):
         """Generate a string to specify a desired variation.
 
-        This method converts a user input string defining a desired solution variation into a valid
+        This method converts an input string defining a desired solution variation into a valid
         string taking into account all existing design properties and project variables, including 
         dependent (non-sweep) properties.
 
-        This is needed because the standard function `GetVariationVariableValue` does not work for obtaining
+        This is needed because the standard method :funct:`GetVariationVariableValue` does not work for obtaining
```

Comment:
```suggestion
        This is needed because the standard method COM function ``GetVariationVariableValue`` does not work for obtaining
```
This is a special one because the COM object (the underlying API) doesn't have any python documentation, so we can't link to this.